gson/src/main/java/com/google/gson/internal/$Gson$Preconditions.java

```diff
@@ -31,6 +31,7 @@
  * @author Inderjeet Singh
  * @author Joel Leitch
  */
+@SuppressWarnings("MemberName") // legacy class name
 public final class $Gson$Preconditions {
```

Comment:
Slightly related: #1744

At least Eclipse IDE has problems with these class names sometimes. On the other hand they make accidental usage by external projects less likely.